### PR TITLE
docs: Readme

### LINKED ISSUE #658: Readme should say exactly how to connect to the current mainnet
## Summary

readme.md should include exact instructions on how to connect to the current gaia mainnet.

## Problem Definition

Users aren't sure how to do it and are asking in Discord

## Proposal

I'll submit a PR asap
____

#### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [x] Contributor assigned/self-assigned

### PR DESCRIPTION
**Pull request by faddat**
Closes: #658 

## Description

Adds an an instant-gratification code snippet to readme.md so that it's always clear how to start a mainnet node. 
______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/main/README.md#merging-a-pr))

### CODE REVIEW

**Review by zmanian — APPROVED** [review]

File: docs/README.md [hunk 1]
```diff
@@ -8,6 +8,26 @@ layout: home
...
 Welcome to the documentation of the **Cosmos Hub application: `gaia`**.
 
+## Join Mainnet
+
+- [Detailed node setup information](./docs/gaia-tutorials/join-mainnet.md)
+
+**Instant Gratification Snippet**
+
+```bash
+git clone https://github.com/cosmos/gaia
```
Comment (alessio) [inline docs/README.md]:
```suggestion
git clone -b v4.0.4 https://github.com/cosmos/gaia
```

**Review by alessio — APPROVED** [review]
I love this! Thanks @faddat :pray:!

File: docs/README.md [hunk 2]
```diff
@@ -8,6 +8,26 @@ layout: home
...
+
+**Instant Gratification Snippet**
+
+```bash
+git clone -b v4.0.4 https://github.com/cosmos/gaia
+make install
+gaiad init chooseanicehandle
+wget https://github.com/cosmos/mainnet/raw/master/genesis.cosmoshub-4.json.gz
+gzip -d genesis.cosmoshub-4.json.gz
+mv mv genesis.cosmoshub-4.json ~/.gaia/config/genesis.json
```
Comment (shahankhatch) [inline docs/README.md]: `mv` is repeated

File: docs/README.md [hunk 3]
```diff
@@ -8,6 +8,26 @@ layout: home
 
 Welcome to the documentation of the **Cosmos Hub application: `gaia`**.
 
+## Join Mainnet
```
Comment (shahankhatch) [inline docs/README.md:11]: Please move these instructions into their own doc and link to it from this readme. There can be a new folder called networks with 'mainnet' and 'testnet' subfolders, and these instructions can be placed there. As well, if you want, you can create this actual script and place it in the networks folder of the root, alongside the other scripts.

**Review by shahankhatch — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 3 above)
Comment (shahankhatch) [reply docs/README.md:11]: Alternatively, I'm happy to accept this PR and do these changes, if you want.

File: docs/README.md (hunk 2 above)
Comment (faddat) [reply docs/README.md]: hey thank you!

File: docs/README.md (hunk 3 above)
Comment (faddat) [reply docs/README.md:11]: I can do it, **but** I think we need a quick start front and center. When I review project code, the very first thing I always want to do is run a node and indeed I judge a bit based on how easy it is. 

An alternative that I considered is a curl | bash type script or a docker image, which would leave the Readme "landing page" less busy but equally functional.

What do you think?

Comment (shahankhatch) [reply docs/README.md:11]: I really like the curl idea! And bonus points for a docker image for connecting to mainnet (can be a separate PR if you wish).

Comment (faddat) [reply docs/README.md:11]: OK, sounds great then!

**Pull request closed without merging**